index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+    <nav>
+        <h1>Dr. Borlaug
+        </h1>
+        <h1>The man who saved a billion lives</h1>
+    </nav>
+
+    <div class="background-image">
```

Comment:
instead of <div> tag use here <section>  .

Reply:
- [ ] please always Remember not to be authoritative.. you are reviewing and suggesting changes not imposing them.

Reply:
Done